packages/theme-chalk/src/input.scss

```diff
@@ -291,7 +291,6 @@
     justify-content: center;
     align-items: center;
     transition: all getCssVar('transition-duration');
-    margin-left: 8px;
```

Comment:
Because of the change, there are parts that work differently than before.

### before

<img width="631" alt="image" src="https://user-images.githubusercontent.com/27342882/224216923-6d2bc7bb-5a35-4388-9e0d-51b7cea5bc3c.png">

### after

<img width="772" alt="image" src="https://user-images.githubusercontent.com/27342882/224217113-abf8bc6c-cb22-4e2a-b989-0f9a113aea66.png">

Reply:
I should have raised the gap to the public section.